### PR TITLE
🤖 Remove `online_editor.ace_editor_language` key

### PR DESCRIPTION
**Pull request by ErikSchierboom**
We've recently switched from Ace to CodeMirror as our editor. This PR removes the now obsolete `.online_editor.ace_editor_language` key.

## Tracking

https://github.com/exercism/v3-launch/issues/40

### CODE REVIEW

**Review by mpizenberg — APPROVED** [review]

**Comment by mpizenberg** [comment]
Should we wait for a configlet update before merge?

**Comment by ErikSchierboom** [comment]
@mpizenberg We'll update configlet later today hopefully. Feel free to wait or merge, whatever you prefer. We might automatically merge these later today anyway, so 🤷

**Pull request merged**
